content/en/testing-guide/_index.adoc

```diff
@@ -1,65 +1,26 @@
-image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-jjzRxbT/0/5267278f/L/_MMR0424-L.jpg[] 
-
-The other hose, the blue hose, is left exposed to the air to provide a sample of the ambient air to the Portacount.
-
-Here are the videos of the various tests performed:
-
-video::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-Kq8Lp8G/0/37bf76f3/1920/IMG_5191-1920.mp4[]
+We came up with a series of tests to ensure that the PAPRa would be performant.  Performant means:
 
-Testing the fit of the mask itself:
+1.  When worn, the wearer received N99 levels of protection. link:testing_march_2021[This page] describes how we did that testing in March of 2021 on both individual fan units and on a tester wearing the entire PAPRa.
+2.  The unit must provide a minimum amount of flow.  Hooded PAPRs require a flow of 6 CFM.  Since the PAPRa is designed to provide airflow for a half facepiece, the flow requirement is 4 CFM, or 113.27 L/min.  As of link:flow_testing_may_2021[May 2021], PAPRas provide 67-92 L/min of airflow, depending on hose configuration, so more work is necessary to bring the units up to this flow specification[1]. 
 
-video::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-QkqNg2x/0/764af364/1920/IMG_5189-1920.mp4[]
+== WARNING
 
-Testing the efficacy of various HEPA filters (takeaway: make sure you have good filters from https://germguardian.com[germguardian.com], as specified in the BOM, or something similar):
+Before you wear a PAPRa in any kind of potentially hazardous situation, it is critical that you test it to ensure that the unit is performing at an adequate level.  We have personally worn these units for many hours and tested these units to ensure an N99 level of performance, but we cannot guarantee that the units you make will perform at the same levels.  We provide no warranty, implicit or explicit, that any such units will provide protection.  Please make sure to test your units accordingly and to your own level of comfort and in compliance with whatever local regulations require.
 
-video::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-GgKRbwK/0/a835042a/1920/IMG_5188-1920.mp4[]
+References:
 
-As you can see, while these tests are promising, they are not yet completely rigorous.  We are currently working on a full PAPR testing system, and will likely also reach out to third party testing labs for further verification of individual units.
+1. link:https://multimedia.3m.com/mws/media/378601O/interpretation-of-inhalation-airflow-measurements.pdf[Interpretation of Inhalation Airflow Measurements for Respirator Design and Testing. L.L. Janssen, N.J. Anderson, P.E. Cassidy, R.A. Weber, and T.J. Nelson. Journal of the International Society for Respiratory Protection, Vol. 22, pp. 122-141] referencing Burgess WA, Reist PC (1969) Supply rates for powered air purifying respirators.   Am Ind Hyg Assoc J 30:1-6. 
```

Comment:
I would convert this to a [footnote](https://docs.asciidoctor.org/asciidoc/latest/macros/footnote/)

Reply:
done

Reply:
looks good